test/fileWatchers-test.ts

```diff
@@ -3,10 +3,11 @@ import {mkdirSync, renameSync, unlinkSync, utimesSync, writeFileSync} from "node
 import {basename, dirname, extname, join} from "node:path/posix";
 import {InternSet, difference} from "d3-array";
 import {temporaryDirectoryTask} from "tempy";
+import {normalizeConfig} from "../src/config.js";
 import {FileWatchers} from "../src/fileWatchers.js";
 import {resolvePath} from "../src/path.js";
 
-describe("FileWatchers.of(root, path, names, callback)", () => {
+describe("FileWatchers.of(root, path, names, callback)", async () => {
```

Comment:
You can’t have an `async` callback to `describe`; any tests after an `await` would be ignored.